src/app/users/create-user.ts

```diff
@@ -38,7 +39,11 @@ export const createUser = async ({
   const phoneNumberValid = checkedToPhoneNumber(phone)
   if (phoneNumberValid instanceof Error) return phoneNumberValid
 
-  const userRaw: NewUserInfo = { phone: phoneNumberValid, phoneMetadata }
+  const userRaw: NewUserInfo = {
+    phone: phoneNumberValid,
+    phoneMetadata,
+    role: getTestAccounts().find(({ phone }) => phone === phoneNumberValid)?.role,
+  }
```

Comment:
We don't need this change for the test accounts we create locally (through tests) but I'm guessing we need it in production?

Looking up test-accounts every time we create a user is not ideal. Can we think of an alternative?

Reply:
Agree not perfect solution - hack is fine though IMO until we implement proper authorization logic. (ie casbin)